packages/block-library/src/comment-date/index.php

```diff
@@ -31,14 +31,14 @@ function render_block_core_comment_date( $attributes, $content, $block ) {
 	$link               = get_comment_link( $comment );
 
 	if ( ! empty( $attributes['isLink'] ) ) {
-		$formatted_date = sprintf( '<a href="%1s">%2s</a>', $link, $formatted_date );
+		$formatted_date = sprintf( '<a href="%1s">%2s</a>', esc_url( $link ), esc_html( $formatted_date ) );
 	}
 
 	return sprintf(
 		'<div %1$s><time datetime="%2$s">%3$s</time></div>',
 		$wrapper_attributes,
-		get_comment_date( 'c', $comment ),
-		$formatted_date
+		esc_attr( get_comment_date( 'c', $comment ) ),
+		esc_html( $formatted_date )
```

Comment:
`$formatted_date` is potentially a link, see above. This would break the link.

Reply:
>  see above

Could you explain what you are refering to?

[`get_comment_date()`](https://developer.wordpress.org/reference/functions/get_comment_date/) returns a string although I guess as it's filterable it could pretty much return anything? 🤷‍♂️